tests/openvino/native/quantization/test_weights_compression.py

```diff
@@ -27,6 +27,7 @@
 from nncf.errors import ValidationError
```

Comment:
Please extend `test_raise_error_with_unsupported_params_for_int8` tests for all backends with `backup_mode` option

Reply:
done